### PR TITLE
Added SpoutWorld.getEntity(int id)

### PR DESCRIPTION
**Pull request by Darsstar**
See https://github.com/SpoutDev/SpoutAPI/pull/104 for API

### CODE REVIEW

File: src/main/java/org/spout/server/SpoutWorld.java [hunk 1]
```diff
@@ -118,6 +114,16 @@
 	 */
 	private final Set<Player> players = Collections.newSetFromMap(new ConcurrentHashMap<Player, Boolean>());
 
+	/**
+	 * A queue of all entities to be acclocated in the entity manager
```
Comment (ZNickq) [inline src/main/java/org/spout/server/SpoutWorld.java]: acclocated? o.O

File: src/main/java/org/spout/server/SpoutWorld.java [hunk 2]
```diff
@@ -25,13 +25,9 @@
  */
 package org.spout.server;
 
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```
Comment (Darsstar) [inline src/main/java/org/spout/server/SpoutWorld.java]: bah, ty sonny

File: src/main/java/org/spout/server/SpoutWorld.java (hunk 1 above)
Comment (SonnyX) [inline src/main/java/org/spout/server/SpoutWorld.java]: accLocated:
A method that retrieves the specified object's current screen location.

File: src/main/java/org/spout/server/SpoutWorld.java (hunk 2 above)
Comment (SonnyX) [inline src/main/java/org/spout/server/SpoutWorld.java]: np ^^

File: src/main/java/org/spout/server/SpoutWorld.java (hunk 1 above)
Comment (ZNickq) [inline src/main/java/org/spout/server/SpoutWorld.java]: I am pretty sure it's supposed to be allocated though, lawl (since it's the spawn Queue ;P)

**Pull request merged**